plugins/aws/s3/bucketEncryptionInTransit.js

```diff
@@ -86,5 +97,98 @@ module.exports = {
+        var bucketLocation;
+
+        for (var key in bucketLocations) {
+            if (bucketLocations[key][bucketName]) {
+                bucketLocation = key;
+                var policy = bucketLocations[key][bucketName];
+                break;
+            }
+        }
+
+        // add the location of the bucket to the config
+        config.region = bucketLocation;
+
+        // create the params necessary for the remediation
+        var params = {};
+        var SecureTransport = {
+            'Sid':'DenyInSecureTransport',
+            'Effect': 'Deny',
+            'Principal': '*',
+            'Action': 's3:*',
+            'Resource': [resource,resource+'/*'],
+            'Condition': {
+                'Bool': {
+                    'aws:SecureTransport': 'false'
+                }
+            }
+        };
+        var policyBody = {
+            'Version': '2012-10-17'
+        };
+        if (policy.err &&
+            policy.err.code === 'NoSuchBucketPolicy'){
+            policyBody['Statement'] = SecureTransport;
+            params = {
+                'Bucket': bucketName,
+                'Policy': JSON.stringify(policyBody)
+            };
+        } else {
+            var policyJson;
+            if (typeof policy.data.Policy === 'string') {
+                policyJson = JSON.parse(policy.data.Policy);
+            } else {
+                policyJson = policy.data.Policy;
+            }
+            policyJson.Statement.push(SecureTransport);
+            params = {
+                'Bucket': bucketName,
+                'Policy': JSON.stringify(policyJson)
+            };
+        }
+
+        var remediation_file = settings.remediation_file;
+
+        remediation_file['pre_remediate']['actions'][pluginName][resource] = {
+            'VersioningConfiguration': 'Suspended',
+            'Bucket': bucketName
+        };
+
+        // passes the config, put call, and params to the remediate helper function
+        helpers.remediatePlugin(config, putCall[0], params, function(err) {
+            if (err) {
+                remediation_file['remediate']['actions'][pluginName]['error'] = err;
+                return callback(err, null);
+            }
+
+            let action = params;
+            action.action = putCall;
+
+            remediation_file['post_remediate']['actions'][pluginName][resource] = action;
+            remediation_file['remediate']['actions'][pluginName][resource] = {
+                'VersioningConfiguration': 'Enabled',
+                'Bucket': bucketName
+            };
+            settings.remediation_file = remediation_file;
+            return callback(null, action);
+        });
+    },
+    rollback: function(config, cache, settings, resource, callback){
+        console.log('Rollback support for this plugin has not yet been implemented');
+        console.log(config);
+        console.log(cache);
+        console.log(settings);
+        console.log(resource);
```

Comment:
can you remove the debug logs

Reply:
i kept that function for place holder otherwise our UTS will fail. if i remove the debug logs eslint will report error saying unused variables.

Reply:
make it a one line console log

Reply:
done